doc/source/installation/post-installation.rst

```diff
@@ -0,0 +1,177 @@
+.. _post-installation:
+
+===============================================================================
+Post-Installation
+===============================================================================
+
+After you have installed |Percona Server|, you may need to do the following:
+
+* Initialize the data directory. Some installation methods, such as ``apt`` or ``yum``, automatically initialize the data directory.
+* Update the ``root`` password. The data directory initialization may
+  update the password.
+* Start the server
+* Configure the server to start on startup.
+* Test access to the server
+* Enable time zone recognition.
+
+Initializing the Data Directory
+-------------------------------
+
+Some installation methods automatically initialize the data directory. If you have installed the server using source distribution or generic binary distribution files, the data directory is not initialized. You only need to run the initialization process after you have first installed the server. 
+
+Run `mysqld` with the `--initialize <https://dev.mysql.com/doc/refman/5.7/en/server-options.html#option_mysqld_initialize>`_ option or the `initialize-insecure <https://dev.mysql.com/doc/refman/5.7/en/server-options.html#option_mysqld_initialize-insecure>`_ option.
+
+Executing `mysqld` with either option does the following:
+
+* Checks for the existence of the data directory
+* Initializes the system tablespace and related structures
+* Creates system tables including grant tables, time zone tables, and   server-side help tables
+* Creates ``root@localhost``
+
+You should run the following steps with the user ID of the ``mysql`` login. 
+
+1. Navigate to the mysql directory. The example uses the default location.
+
+.. code-block:: bash
+
+    $ cd /usr/local/mysql
+
+2. Create a directory for the mysql files. The `secure_file_priv <https://dev.mysql.com/doc/refman/5.7/en/server-system-variables.html#sysvar_secure_file_priv>`_ can use the directory as a value.
+
+.. code-block:: bash
+
+    $ mkdir mydata
+
+Grant the appropriate permissions to the directory.
+
```

Comment:
Describing the `appropriate permissions ` would be more helpful

Reply:
Added a table